src/applications/disability-benefits/526EZ/components/createDisabilityIncreaseApplicationStatus.jsx

```diff
@@ -2,11 +2,29 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 
-const disabilityForms = new Set(['21-526EZ']);
+import backendServices from '../../../../platform/user/profile/constants/backendServices';
+import formConfig from '../config/form';
+
+const { formId } = formConfig;
+const { EVSS_CLAIMS } = backendServices;
+const disabilityForms = new Set([formId]);
 
 export default function createDisabilityIncreaseApplicationStatus(store) {
   const root = document.getElementById('react-applicationStatus');
-  if (root) {
+  const state = store.getState();
+  const { user: { profile: { savedForms, services } } } = state;
+  const hasEVSSClaimsService = !!(services.find(service => service === EVSS_CLAIMS));
+  const hasSavedForm = !!(savedForms.find(form => form === formId));
+  const shouldGateForm = !hasEVSSClaimsService && hasSavedForm;
+  if (shouldGateForm) {
+    ReactDOM.render((<div itemProp="steps" itemScope itemType="http://schema.org/HowToSection">
+      <h3 itemProp="name">Ready to apply?</h3>
+      <div itemProp="itemListElement">
+        <div className="usa-alert usa-alert-error no-background-image">Sorry, our system is temporarily down while we fix a few things. Please try again later.</div>
```

Comment:
Same concern here as above; should we display the "system is temporarily down" message if the user doesn't have `evss_claims`?

Reply:
Unless the user has saved forms and doesn't have evss_claims as a servce, the wizard is provided, I'm not aware that we want to block access to information, only to accessing the form.

Reply:
Right, I think the gating logic is solid, but the error message is misleading.

@kreek What are the possible reasons a user won't have the `evss_claims` service?